experimental/ni_coq/oak-ni-coq.opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "2.0"
+maintainer: "aferr@google.com"
```

Comment:
Jade, do you want to also be listed as a maintainer? (Only if you want).

Reply:
All the opam packages I looked at as examples seem to have only one name (I'm not sure if this is a rule or just convention). I put yours since you're the main project architect, but if you don't want to be on the hook for opam maintenance we can change it. Since this opam package configuration is just a dummy for the build system, I think no one is likely to receive maintenance requests!